oex_plugin/openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -124,4 +125,18 @@ def _update_additional_extended_profile_fields_if_needed(data, user):
     """
     if 'english_proficiency' in data:
         user.extended_profile.english_proficiency = data['english_proficiency']
-        user.extended_profile.save()
+    if 'organization' in data:
+        if org_data := data['organization']:
```

Comment:
I think we can get rid of one indent level here:
```suggestion
   if org_data := data.get('organization'):
```

Reply:
discussed and decided to leave as it is because we need to handle case when organization key is present but its value is falsy